README.md

```diff
@@ -89,11 +89,11 @@ The next two sections presume you are using the out-of-the-box `.mf.config.json`
 
 Mock Firebolt supports these Firebolt SDKs: **core**, **manage**, **discovery**.
 
-By default, only the core SDK is enabled, meaning if you try to make calls to methods in the other SDKs, Mock Firebolt will thow an error.
+By default, only the core/manage OpenRPC retrieved from HTTP is enabled
 
 This default mode is appropriate for app developers developing third-party content apps that don't need (nor get) the permissions necessary to use the other SDKs.
```

Comment:
This statement really isn't true any more. Since the one OpenRPC file has "everything", Mock Firebolt will respond to manage SDK calls, which generally third-party content parties will use. So I think I would just take this paragraph out.